roam_to_git/scrapping.py

```diff
@@ -300,6 +301,9 @@ def signin(browser: Browser, config: Config, sleep_duration=1.):
             logger.debug("Sign In Failed")
             logger.trace("StaleElementReferenceException: Retry getting the email field")
             time.sleep(1)
+    else:
+        logger.debug("Sign in attempt exceeded 5")
```

Comment:
Can you raise an Error instead of logging it to debug and exiting?

Reply:
I am raising a connection error for this now